addons/web/static/src/legacy/scss/dropdown.scss

```diff
@@ -54,3 +54,11 @@
         }
     }
 }
+
+.o_dropdown_menu {
```

Comment:
I think this code should be here ;)
https://github.com/odoo/odoo/blob/master/addons/web/static/src/core/dropdown/dropdown.scss#L24